src/hotspot/share/opto/compile.hpp

```diff
@@ -631,6 +632,10 @@ class Compile : public Phase {
   bool              has_monitors() const         { return _has_monitors; }
   void          set_has_monitors(bool v)         { _has_monitors = v; }
 
+  void          push_monitor() { _max_monitors++; }
+  void          reset_max_monitors() { _max_monitors = 0; }
+  uint          max_monitors() { return _max_monitors; }
```

Comment:
The prevailing style in this file appears to have some indenting after the ')'
and before the '{'. It's somewhat inconsistent as to how much, but mostly
more than a single space.